projects/angular-cld/src/lib/cloudinary-image.component.ts

```diff
@@ -174,6 +177,25 @@ export class CloudinaryImage
   }
 
   accessibilityModeHandler() {
-    return this.cloudinary.url(this.publicId, {transformation: [this.options, accessibilityEffect[this.accessibility]]});
+    return this.cloudinary.url(this.publicId, { transformation: [this.options, accessibilityEffect[this.accessibility]], analyticsOptions: this.options['analyticsOptions'] });
+  }
```

Comment:
Could this be moved to the JS-Core package?

Reply:
That's not part of the pr.. the accessibility handler has always been here. If we wish to move it should be moved in nir's pr for the advanced image component